python/lsst/pipe/tasks/make_direct_warp.py

```diff
@@ -316,6 +336,36 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
 
         visit_summary = inputs["visit_summary"] if self.config.useVisitSummaryPsf else None
 
+        # Can this go in the run method?
```

Comment:
Should be doable, given that the `run` method takes `DeferredDatasetHandles`, and you can get data IDs from those.